app/assets/javascripts/application.js

```diff
@@ -38,6 +38,7 @@
 //= require ./angular-libs/guides/directives/guides.formChecker.directive.js
 
 //= require ./angular-libs/ngs3upload
+//= require ./angular-libs/angular-socialshare
```

Comment:
Instead of adding the library to our repository (so we don't track our own version of the angular-socialshare library) could you use rails-assets? https://rails-assets.org/#/components/angular-socialshare

There's examples of how to do this in the gemfile for other angular libraries. Then you can update application.js file here.